ullyses/coadd.py

```diff
@@ -9,12 +9,12 @@
 from datetime import datetime as dt
 
 from ullyses_utils.ullyses_config import CAL_VER
-
+from ullyses.combine_header_keys import KeyBlender
 # coadd data
 #
 
 STIS_NON_CCD_DETECTORS = ['FUV-MAMA', 'NUV-MAMA']
-class SegmentList:
+class SegmentList(KeyBlender):
```

Comment:
This seems a bit extreme - I like to think of the SegmentList class as the class that handles coaddition of spectra, and now you're deriving it from a separate class that has functionality over all Ullyses products.  I get that you want to be able to do self.combine_keys(), but it feels like adding this to the SegmentList base class is rather too high a price to pay in that it overcomplicates what should be a simple base class. 
I'd suggest keeping the current combine_keys method in the segmentlist class, and if you want the ullyses_segmentlist class to have different, more extensive behaviour that includes FUSE, LCOGT, WFC3, VLT, have it derive from KeyBlender and SegmentList.
The rule should be that importing a SegmentList from coadd shouldn't require importing any other modules in the Ullyses package (technically, not even CAL_VER) to preserve the lack of dependency of hasp on ullyses